src/application.rs

```diff
@@ -147,6 +147,7 @@ impl Application {
             .modal(true)
             .authors(vec!["Marcus Behrendt".into()])
             .artists(vec!["Marcus Behrendt".into()])
+            .artists(vec!["Allaeddine Boulefaat".into()])
```

Comment:
You should put your name directly into the `vec`. Else, you overwrite my name. :-P

```rust
.artists(vec!["Marcus Behrendt".into(), "Allaeddine Boulefaat".into()])
```

I would suggest you change the contents and messages of the two commits with an interactive rebase. So we have a clean history. The commit messages should start with a capital letter and should be in imperative style ("Add" instead of "added").

The process is like

```shell
$ git rebase -i HEAD~2
```

Then you'll see this in a text editor 
```
pick 7d892b2 added application icons
pick 44708a1 added myself to the artists array.
```

Change the two `pick`s to `r` (reword) and `e` and (edit)
```
r 7d892b2 added application icons
e 44708a1 added myself to the artists array.
```

Save and leave the editor. Then you are prompted to alter the message of the first commit. After you have done it, you are in edit mode for the 2nd commit. Here you fix the artist vec. Then, add your changes to the staging area with
```shell
$ git add -A
```

Finally, finish the rebase process
```
$ git rebase --continue
```

At last, force push
```
$ git push --force
```